lib/components/Button/Button.js

```diff
@@ -10,13 +10,17 @@ const VARIANT = {
   NO_OUTLINE: 'NO_OUTLINE',
 };
 
+const VARIANT_DEFAULT = VARIANT.SECONDARY;
+
 const SIZE = {
   LARGE: 'LARGE',
   MEDIUM: 'MEDIUM',
   SMALL: 'SMALL',
   SQUARE: 'SQUARE',
 };
 
+const SIZE_DEFAULT = SIZE.MEDIUM;
```

Comment:
Is the extra indirection necessary? I'd make a case that `defaultProps` is implicitly enough to understand the context

Reply:
Not for this component, but I've used the defaults in the past to implicitly determine aria values (i.e. main menu)